contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.4.0--3.5.0.sql

```diff
@@ -59,6 +59,36 @@ WHERE t5.contype = 'p'
   AND ext.dbid = sys.db_id();
 
 
+ALTER FUNCTION sys.parsename(sys.VARCHAR, INT) RENAME TO parsename_deprecated_in_3_5_0;
```

Comment:
Lets use defensive coding here. Use try..catch block to achieve this

Reply:
Added Exception statement to use defensive code here.